### ISSUE TITLE
Jfix icon on Maven Repository

### ISSUE THREAD

**Issue by PT400C** [ru-fix/jfix-stdlib#49]
Hey,

having used your library for quite some time now I randomly noticed that it has an icon on mvnrepository.com. How does that work? E.g. my projects don't have that icon on mvncentral, though they're written in Java / Kotlin too.

![grafik](https://user-images.githubusercontent.com/20016523/82016923-41148480-9682-11ea-8318-3f168ae4b3b4.png)

Thanks in advance

PT400C

**Comment by swarmshine** [comment]
Hello. 
In order to be able to upload artifacts to maven-central though https://oss.sonatype.org/ you have to pass registration within https://issues.sonatype.org/  
Usually you create a request to register a new root package name for your artifacts in 
https://issues.sonatype.org/ ticket. 
This package name can be domain name of your's company web site or address of your github profile. 
You can use web site of your company and provide logo picture during registration process. Then this logo will popup in https://mvnrepository.com/

**Comment by PT400C** [comment]
@swarmshine Thank you so much for this detailed explanation! This really helps!

**Comment by PT400C** [comment]
@swarmshine Sorry to bother you once again, but it seems sonatype isn't involved in the operation of mvnrepository. Thus issueing this request on their JIRA tracker doesn't allow me to hand a picture to them. Did this change in the last months / years or did I understand you wrongly?

**Comment by swarmshine** [comment]
Sorry for misleading you into wrong direction. 
I've checked our old e-mails with sonatype support team. Looks like you are right, there was no logo submitted from our company.
Also I found this stackoverflow question:
https://stackoverflow.com/questions/40197177/how-to-create-a-logo-for-mvnrepository-artifact
And found that the logo that currently exist in https://mvnrepository.com/ is a logo that WAS available at https://github.com/ru-fix/  during our first artifact publication.  After that we switched logo in github to new one. 
So based on the answer from stackoverflow, seems like https://mvnrepository.com/ tries to get logo image from SCM URL path from project pom.xml when you first time deploy it in repository.
